<launch>
    <node pkg="omniGKF_control" type="gkf_server_node" name="omni_gkf_usb_server" >
        <param name="omni_gkf_usb_server/port" value="/dev/ttyACM0" />
        <param name="omni_gkf_usb_server/baudrate" value="115200" />
        <param name="gkf_k_vel" value="1.0" />
        <param name="gkf_k_pos" value="1.0" />
    </node>
    <!-- <node pkg="OmniGKF_control" type="test_omni1.py" name="test_omni1"  /> -->
</launch>
